libs/base/ui/form/password/src/password-input.component.ts

```diff
@@ -62,15 +64,15 @@ export class PasswordInputComponent
   @property({ type: Boolean }) visible?: boolean;
 
   @state()
-  minLengthError = false;
+  minLengthError = true;
   @state()
-  maxLengthError = false;
+  maxLengthError = true;
   @state()
-  minUppercaseCharsError = false;
+  minUppercaseCharsError = true;
   @state()
-  minNumbersError = false;
+  minNumbersError = true;
   @state()
-  minSpecialCharsError = false;
+  minSpecialCharsError = true;
```

Comment:
So we changed default state?

Reply:
Before hydration, these values are used to determine the active state and on the server, it showed up as `active` without running validation if you leave it `true`. But we wouldn't really have any input to test it on anyways since that all happens on the client and not the server side. I made with the assumption that the text field is empty from the server's perspective, so nothing is valid.

Unless there's a more preferable approach to this.